File: FlappyJournal/server/add-more-debug.js
import fs from 'fs';

const filePath = './enhanced-dual-consciousness-ws.js';
let content = fs.readFileSync(filePath, 'utf8');

// Add debug after consciousness processing
content = content.replace(
  'const consciousnessResult = await consciousness.process(userMessage',
  `console.log('Starting consciousness processing...');
          const consciousnessResult = await consciousness.process(userMessage`
);

// Add debug after getting result
content = content.replace(
  '// Create consciousness context for AI systems',
  `console.log('Consciousness result:', consciousnessResult);
          
          // Create consciousness context for AI systems`
);

// Add debug for response sending
content = content.replace(
  'ws.send(JSON.stringify({',
  `console.log('Sending response to client');
          ws.send(JSON.stringify({`
);

fs.writeFileSync(filePath, content);
console.log('Added more debug logging');

File: FlappyJournal/server/add-more-debug.py
#!/usr/bin/env python3

with open('enhanced-dual-consciousness-ws.js', 'r') as f:
    content = f.read()

# Add logging before the try block for AI responses
content = content.replace(
    "// Get responses from both AI systems",
    """console.log('About to get AI responses...');
          // Get responses from both AI systems"""
)

# Add logging after consciousness processing
content = content.replace(
    "console.log('Creative insight:', creativeResult.insight);",
    """console.log('Creative insight:', creativeResult.insight);
          console.log('Now proceeding to AI response generation...');"""
)

with open('enhanced-dual-consciousness-ws.js', 'w') as f:
    f.write(content)

print("More debug logging added")
